pyproject.toml

```diff
@@ -30,19 +30,26 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
-dev = ["black", "pytest", "pytest-cov", "flake8", "mypy", "pre-commit"]
 converters = ["qiskit"]
 
-[tool.hatch.envs.tests]
-features = [
-  "dev",
-  "converters",
+[tool.hatch.envs.default]
+dependencies = [
+  "pytest",
+  "pytest-cov",
+  "pytest-mypy",
+  "pytest-xdist",
+  "nbconvert",
+  "ipykernel",
+  "pre-commit",
+  "flake8",
+  "black",
+  "isort",
+  "mypy"
 ]
```

Comment:
I think the old behavior is better . Users shouldn't have to install pytest, pre-commit etc

Reply:
ok i see! i am not done with the PR yet. i just wanted to improve the overall code quality by adding these tools so a dev can easily use them

Reply:
Yes, but what @awennersteen was pointing out is that it is better to keep them in a separate `tests` environment rather than adding them to the default. Then you can just do something like:
```
python -m hatch --env tests run pytest
```

Reply:
changed it back